lib/utilities/mouseDownDrag.js

```diff
@@ -93,6 +93,9 @@ var onMouseMove = function(name, pseudoDragCallback, scaleFactor, event) {
  * @returns {void}
  */
 var onMouseUp = function(name, pseudoDragEndCallback, event) {
+  if (!this.isMounted()) {
+    return;
+  }
```

Comment:
A "drag" is anything between a mouse down and a mouse up. Between that you could have opened a new container, unmounting the previous Graph.